### PR TITLE
Feat : added wishlist page

### PR DESCRIPTION
**Pull request by TusharBhandirge**
Added Wishlist page of my E-com app

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
It would be great if you could add a preview link maybe.

**Comment by TusharBhandirge** [comment]
Preview Link  : https://wishlist-preview.netlify.app/wishlistpage/wishlistpage

File: wishlistPage/wishlistPage.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                    <a class="link" href="https://www.linkedin.com/in/tushar-bhandirge-300249118/">
+                        <img class="icon-image" src="./Image_resources/icons8-linkedin-30.png" />
+                    </a>
+               
+                    <a class="link" href="https://twitter.com/TusharBhandirge?s=09">
+                        <img class="icon-image" src="./Image_resources/icons8-twitter-30.png" />
+                    </a>
+               
+                    <a class="link" href="https://github.com/TusharBhandirge?tab=repositories">
+                        <img class="icon-image" src="./Image_resources/icons8-github-30.png"/>
```
Comment (preethasri) [inline wishlistPage/wishlistPage.html]: hii tushar check with your image source link for footer. other than that UI looks good👍

**Review by preethasri — APPROVED** [review]

File: wishlistPage/wishlistPage.html (hunk 1 above)
Comment (TusharBhandirge) [reply wishlistPage/wishlistPage.html]: okay i'll correct this image source. Thank!

**Pull request merged**